doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -116,6 +116,7 @@ Common Application Framework (CAF)
 Added:
 
 * :ref:`caf_preview_sample` sample.
+* Added :ref:`caf_buttons_pm_keep_alive`.
```

Comment:
Added word is typed twice now

Reply:
Well - it is the same like it is done in nRF Desktop subtitle. We should decide which version we would use.

Reply:
Already cleaned.